client/src/tests/Cart.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
Add 1 more test to ensure it prints to correct total price with some dummy cart items as updated in your change as well!

Reply:
+1